### PR TITLE
Split privileged into privileged and normandy-privileged

### PR DESCRIPTION
**Pull request by wagnerand**
Supports https://github.com/mozilla-extensions/xpi-manifest/pull/67.

### CODE REVIEW

**Comment by wagnerand** [comment]
@escapewindow r?

**Review by escapewindow — APPROVED** [review]
Thanks! This can land at any time. It may make sense to wait until the xpi-manifest and shipit code is close to landing, to minimize the window where CI and releases have different sets of signing types.

File: .taskcluster.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ tasks:
     - $let:
           # XXX Set to `true` for private repos
           privateRepo: false
-          # XXX use "privileged", "system", or "mozillaonline-privileged" to
-          #     enable signing on push/PR. Note, this only works for public
-          #     repos for now. Release builds off shipit will get signing for
-          #     both public and private repos.
+          # Use `system` for system add-on,
+          # `hosted-privileged` for AMO or self-hosted privileged add-on,
```
Comment (escapewindow) [inline .taskcluster.yml]: We'll probably want `hosted` to agree with whatever we name it in `xpi-manifest`.

File: taskcluster/xpi_taskgraph/signing.py [hunk 2]
```diff
@@ -19,9 +19,11 @@
 transforms = TransformSequence()
 
 FORMATS = {
-    "privileged": "privileged_webextension",
-    "system": "system_addon",
+    "hosted-privileged": "privileged_webextension",
```
Comment (escapewindow) [inline taskcluster/xpi_taskgraph/signing.py]: Here too.

**Review by escapewindow — APPROVED** [review]
With this change, we probably don't have to go through and update all the normandy-privileged source repos' .taskcluster.yml files. We can, but it's not as important.

**Pull request merged**